### PR TITLE
[All Hosts] (sideloading) instructions for autotrusting with a RegEx …

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
…script

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[44097a1](https://github.com/OfficeDev/office-js-docs-pr/commits/44097a1c77162342c169be98c2fa8c8b03870349)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/autotrust-catalog/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912060012185556-1478%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=EuoJ43ZWe1I5fPNBgsc8KV2D5dMvi7TupGFlH7JtskQ%3D&st=2019-12-06T00%3A08%3A32Z&se=2020-01-06T00%3A13%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[54b5ef8](https://github.com/OfficeDev/office-js-docs-pr/commits/54b5ef804be557cb6d3f4a1aeb0639df2e2b3d5e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/autotrust-catalog/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912060014115385-1478%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=kLCuat9PnHmMhGQjSHZ5C27b9pn6W%2FZB5AeOnI9honM%3D&st=2019-12-06T00%3A10%3A22Z&se=2020-01-06T00%3A15%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[c7f104f](https://github.com/OfficeDev/office-js-docs-pr/commits/c7f104f69f319cbbe3756a6448f52fe52a91bc6b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/autotrust-catalog/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912060016371045-1478%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=i%2FXhnNviF0AAHVqAOnRzGv%2FCNEKGIbdCZTfaDCQ92nQ%3D&st=2019-12-06T00%3A12%3A51Z&se=2020-01-06T00%3A17%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md [hunk 1]
```diff
@@ -63,10 +65,43 @@ The following video walks you through the process of sideloading your add-in in
...
+
+	```
+	Windows Registry Editor Version 5.00
+	
+	[HKEY_CURRENT_USER\Software\Microsoft\Office\16.0\WEF\TrustedCatalogs\{-random-GUID-here-}]
+	"Id"="{-random-GUID-here-}"
+	"Url"="\\\\-share-\\-folder-"
+	"Flags"=dword:00000001
+	```
+3. Use one of the many online GUID generation tools, such as [GUID Generator](https://guidgenerator.com/), to generate a random GUID, and replace "-random-GUID-here-" *in both places* with the GUID. (The enclosing `{}` symbols should remain.)
```
Comment (kbrandl) [inline docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md]:
```suggestion
3. Use one of the many online GUID generation tools, such as [GUID Generator](https://guidgenerator.com/), to generate a random GUID, and within the TrustNetworkShareCatalog.reg file, replace the string "-random-GUID-here-" *in both places* with the GUID. (The enclosing `{}` symbols should remain.)
```

File: docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md [hunk 2]
```diff
@@ -63,10 +65,43 @@ The following video walks you through the process of sideloading your add-in in
...
+
+4. Replace the `Url` value with the full network path to the folder that you [shared](#share-a-folder) previously. (Note that the `\` characters must be doubled.) If you failed to note the folder's full network path when you shared the folder, you can get it from the folder's **Properties** dialog window, as shown in the following screenshot.
```
Comment (kbrandl) [inline docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md]:
```suggestion
4. Replace the `Url` value with the full network path to the folder that you [shared](#share-a-folder) previously. (Note that any `\` characters in the URL must be doubled.) If you failed to note the folder's full network path when you shared the folder, you can get it from the folder's **Properties** dialog window, as shown in the following screenshot. 
```

File: docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md [hunk 3]
```diff
@@ -63,10 +65,43 @@ The following video walks you through the process of sideloading your add-in in
...
+	"Url"="\\\\-share-\\-folder-"
+	"Flags"=dword:00000001
+	```
+3. Use one of the many online GUID generation tools, such as [GUID Generator](https://guidgenerator.com/), to generate a random GUID, and replace "-random-GUID-here-" *in both places* with the GUID. (The enclosing `{}` symbols should remain.)
+
+4. Replace the `Url` value with the full network path to the folder that you [shared](#share-a-folder) previously. (Note that the `\` characters must be doubled.) If you failed to note the folder's full network path when you shared the folder, you can get it from the folder's **Properties** dialog window, as shown in the following screenshot. 
 
+    ![folder Properties dialog with the Sharing tab and network path highlighted](../images/sideload-windows-properties-dialog-2.png)
+	
+5. The file should look like the following. Save it.
```
Comment (kbrandl) [inline docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md]:
```suggestion
5. The file should now look like the following. Save it.
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[caaa477](https://github.com/OfficeDev/office-js-docs-pr/commits/caaa4775680f1a1178ee42b4c4a086cad7a0e190)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/autotrust-catalog/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912061746410494-1478%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=PhWGOl4ZgMQX9N%2Fn4nzASCxLhMpkqL%2BbCHGVikjSgrE%3D&st=2019-12-06T17%3A42%3A56Z&se=2020-01-06T17%3A47%3A56Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[3d3efaf](https://github.com/OfficeDev/office-js-docs-pr/commits/3d3efafb0fb71209558e8788cf53de8694083d50)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/autotrust-catalog/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C12%5C6%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201912061747000777-1478%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=1acXQtmY%2F44Mojvuj8E6T2usGDvkvLUNftsctcquCmQ%3D&st=2019-12-06T17%3A44%3A17Z&se=2020-01-06T17%3A49%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**